permissions/rules/za.js

```diff
@@ -0,0 +1,129 @@
+const CanCan = require('cancan');
```

Comment:
Compared against https://benetech.app.box.com/file/786400222972

All rules seems to be ok!